packages/core/crates/core-types/src/acknowledgement.rs

```diff
@@ -81,20 +82,32 @@ impl BinarySerializable for Acknowledgement {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
 pub enum AcknowledgedTicketStatus {
     Untouched,
     BeingRedeemed { tx_hash: Hash },
```

Comment:
What is the `tx_hash` useful for if it is ever set to `0x0...` only?

Reply:
It's currently used for bookeeping only, reserved for future use.